src/configuration/routes.md

```diff
@@ -115,6 +115,85 @@ https://blog.example.com.sprint-7onpvba-tvh56f275i3um.eu-2.platformsh.site/
 
 If your project involves only a single apex domain with one app or multiple apps under subdomains, it's generally best to use the `{default}` placeholder.  If you are running [multiple applications](/configuration/app/multi-app.md) on different apex domains then you will need to use a static domain for all but one of them.
 
+## Route identifiers
+
+All routes defined on an environment will be available in the application in the `PLATFORM_ROUTES` environment variable.  That value is a base64-encoded JSON string, which can be extracted in the language of your choice to look up the effective routes that were generated.
+
+"Effective route" here means that all placeholders will be replaced with the appropriate domain name, and depending on your configuration additional route entries may be generated as well (such as the automatic HTTP->HTTPS redirect).  That means the domains will vary from one environment to another.  To make it easier to locate specific routes in a standardized fashion you may also specify an `id` on each route that is stable across all environments.  You may also mark one, and only one route as `primary`.  The primary route may be special cased in the interface but has no impact on the runtime environment.
+
+For example, consider this `routes.yaml` configuration:
+
+```yaml
+"https://site1.{default}/":
+    type: upstream
+    upstream: 'site1:http'
+
+"https://site2.{default}/":
+    type: upstream
+    id: 'the-second'
+    upstream: 'site2:http'
+```
+
+This example defines two routes, on two separate subdomains, pointing at two separate app containers.  (They could just as easily be pointing at the same container for our purposes).  On a branch named `test`, the routes array in PHP would look like this:
```

Comment:
If you're naming the array "routes," I'd format it as `routes`. If you're just generally referring to an array for routes, then I think "route array" would be a better phrasing.